recipes/sony_custom_layers/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sony_custom_layers" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sony/custom_layers/archive/v{{ version }}.tar.gz
+  sha256: 43a58a319484169daaeebb7a3a5079faf8a6fbffe3748968cd68754671121ffa
+
+build:
```

Comment:
I think this can be `noarch: python` since it doesn't appear to be building any C-extensions:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python